services/pom.xml

```diff
@@ -101,6 +101,10 @@
             <scope>provided</scope>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-common</artifactId>
+        </dependency>
```

Comment:
Do we know why Netty is not included in 24 already?

Reply:
It is brought in transitively via a metrics dependency in the later branches.

Reply:
@shawkins Thanks for clarifying.

This should be safe to do as we use the dependency in other places as well.